security-framework/src/item.rs

```diff
@@ -132,7 +153,7 @@ impl ItemSearchOptions {
     ///
     /// If this is not called, the default limit is 1.
     #[inline(always)]
-    pub fn limit(&mut self, limit: i64) -> &mut Self {
```

Comment:
This is a breaking change. Maybe you could change it to `impl Into<Limit>` and implement `From<i64> for Limit`?

Reply:
Oh, that's a great idea! Added. It should be a non-breaking change now.